bin/tests

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+cargo test --verbose
```

Comment:
Convenience scripts, we can skip if you prefer.

Reply:
Reasoning for them: never needing to wonder how to do an action in a given project when moving between projects or when new people contribute.

Reply:
I don't have a strong opinion. I've seen Makefiles used for this purpose as well, but we can always clean up if things get unwieldy.